src/xrpld/app/tx/detail/Clawback.cpp

```diff
@@ -197,7 +197,7 @@ preclaimHelper<MPTIssue>(
 TER
 Clawback::preclaim(PreclaimContext const& ctx)
 {
-    AccountID const issuer = ctx.tx[sfAccount];
+    AccountID const issuer = ctx.account;
```

Comment:
Should be const&. Please check everywhere where an alias variable is used for `ctx.account`. It should be `AccountID const&`.